src/osd/ECBackend.cc

```diff
@@ -2220,24 +2163,10 @@ bool ECBackend::RMWPipeline::try_reads_to_commit()
   }
 
   return true;
-  }();
 }
 
 bool ECBackend::RMWPipeline::try_finish_rmw()
 {
-  return [
-    get_parent=[this] (auto...) { return this->get_parent(); },
-    get_osdmap=[this] (auto...) { return this->get_osdmap(); }, // get_parent() basically
-    // important: get_parent()->send_message_osd_cluster()
-    &waiting_reads=this->waiting_reads,
-    &waiting_commit=this->waiting_commit,
-    &pipeline_state=this->pipeline_state,
-    &cache=this->cache,
-    &completed_to=this->completed_to,
-    &committed_to=this->committed_to,
-    &tid_to_op_map=this->tid_to_op_map,
-    cct=(CephContext*)nullptr
-  ] {
```

Comment:
Can we just squash this into the prior commit?  It's not particularly useful for review.

Reply:
Sure, will squash.

Reply:
Squashed.